app/components/DR/ButtonUpDrawer.js

```diff
@@ -9,6 +10,8 @@ import {
   View,
 } from 'react-native';
 
+import languages from '../../locales/languages';
```

Comment:
Please use **withTranslation** to use the same instance of i18n and not created new ones, so the label can change along with the others.
```suggestion
import { withTranslation } from 'react-i18next';
```

Reply:
Do you recommend it? To do that I will need to change all the code because is a class component and `react-i18next` is only for React Hook

Reply:
BottonUpDrawer is a class component, and the package `react-i18next` is only for React Hooks componenets

Reply:
no, withTranslation works with class component